### PR TITLE
[T4A4][T11-B2]Pankaj Bhootra

### PR DESCRIPTION
**Pull request by PankajB1997**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -25,6 +27,8 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.sequenceNumber = Person.nextSequenceNumber;
+        Person.nextSequenceNumber++;
```
Comment (xdshang) [inline src/seedu/addressbook/data/person/Person.java:31]: Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (xdshang) [inline src/seedu/addressbook/data/person/Printable.java:3]: Need a header comment for this interface.

File: src/seedu/addressbook/data/person/Person.java [hunk 3]
```diff
@@ -83,5 +87,16 @@ public int hashCode() {
...
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+   public String getPrintableString(Printable... printables){
+	   String result = "";
+	   for (Printable p: printables){
+		   result += p.getPrintableString() + ", ";
+	   }
+	   result = result.substring(0, result.length()-1) + ".";
```
Comment (xdshang) [inline src/seedu/addressbook/data/person/Person.java:99]: Instead of truncating the string, you can try `String.join(...)`.

**Comment by xdshang** [comment]
@PankajB1997 Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/person/Person.java (hunk 1 above)
Comment (PankajB1997) [reply src/seedu/addressbook/data/person/Person.java:31]: Hi, I did exactly that when creating a new branch. But these changes from another branch still showed up on the new branch

**Pull request closed without merging**